.gitmodules

```diff
@@ -0,0 +1,9 @@
+[submodule "compose/frameworks/support"]
+	path = compose/frameworks/support
+	url = https://github.com/JetBrains/androidx
+[submodule "compose/golden"]
+	path = compose/golden
+	url = https://android.googlesource.com/platform/frameworks/support-golden
+[submodule "compose/external/doclava"]
```

Comment:
Why not https://github.com/JetBrains/doclava-mirror? It have simplified JDK8 tools.jar story.

Reply:
tools.jar is not only needed for doclava, but also for [compiler-hosted](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/compiler/compiler-hosted/integration-tests/build.gradle;l=44;drc=4c006d02dcb96771d6948918b56031ed20830f28)

Reply:
So we could reused for it as well, couldn't we?

Reply:
We can, but instead of  `JAVA_TOOLS_JAR` we will come with `COMPOSE_DESKTOP_GITHUB_BUILD` again, which is the same thing, just not default for androidx :).

Besides we should publish CL to Google, which takes mind/time resources :)